h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -274,6 +272,45 @@ protected void setupLocal() {
       }
     }
 
+    /**
+     * @return Number of connections to an SQL database to be opened on a single node.
+     */
+    int getMaxConnectionsPerNode() {
+      int conPerNode;
+      final String userDefinedMaxConnections = System.getProperty(MAX_USR_CONNECTIONS_KEY);
+      if (userDefinedMaxConnections != null) {
```

Comment:
useless condition: use case already handled in the try-catch below

Reply:
Great catch, thanks. Removed.